lib/puppet/functions/compliance_markup/loaded_maps.rb

```diff
@@ -1,33 +1,25 @@
 Puppet::Functions.create_function(:'compliance_markup::loaded_maps') do
-  def initialize(closure_scope, loader)
+  def loaded_maps()
     filename = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/puppetx/simp/compliance_mapper.rb'
     self.instance_eval(File.read(filename),filename)
-    super(closure_scope, loader)
-  end
-  def loaded_maps()
-    retval = nil
+
     profile_compiler = compiler_class.new(self)
     profile_compiler.load  do |k, default|
```

Comment:
This `load` has not been passed the options with :aux_dirs,  :data_dir, and 'mode' keys.  The first two keys allow the user to load compliance YAML files from a different directory.

Reply:
That's fine. We'll need to flesh out that capability at a later time. It is no longer presented to the end user and is basically an internal debugging/testing tool